plugins/harness-ci-cd/README.md

```diff
@@ -99,6 +100,26 @@ const serviceEntityPage = (
 
 4. Add required harness specific annotations to your software component's respective `catalog-info.yaml` file. Here is an example: https://github.com/harness/backstage-plugins/blob/main/examples/catalog-harness-cicd.yaml
```

Comment:
Should we update https://github.com/harness/backstage-plugins/blob/main/examples/catalog-harness-cicd.yaml with the new one as well? Or maybe keep old and new files.

Reply:
Keeping old and new makes more sense.